### PR TITLE
Some info about me

### PR DESCRIPTION
**Pull request by MJ23M**
@Malgin Can`t add you as reviewer

### CODE REVIEW

**Comment by MJ23M** [comment]
111

File: students.txt [hunk 1]
```diff
@@ -13,6 +13,9 @@ List of students
 6. Vladimir S.
 7. Daniil I.
 8. Daria P. 
-9. Max R.
+9. Max Rakovskyi
+Love videogames, especially Dota2. I like playing basketball and travelling to the mountains.
```
Comment (vit-lebediev) [inline students.txt]: `to the mountains` -> `to mountains`
And please, start each sentence from a new line.

**Review by vit-lebediev — CHANGES_REQUESTED** [review]
In general, looks good. Just a few changes, and can be merged.

**Review by vit-lebediev — CHANGES_REQUESTED** [review]
Good, but still, I've asked to change the phrase from `to the mountains` to `to mountains`. You should use definite article 'the' when you speak about some specific mountains, not mountains in general.

**Comment by MJ23M** [comment]
Add changes

**Review by vit-lebediev — APPROVED** [review]

**Comment by vit-lebediev** [comment]
Looks good. :shipit:

**Pull request merged**